prompt.py

```diff
@@ -224,11 +224,13 @@ def do_create(self, arguments):
 
                 print("Wallet %s " % json.dumps(self.Wallet.ToJson(), indent=4))
 
-                contract = self.Wallet.GetDefaultContract()
-                if contract is not None:
+                try:
+                    contract = self.Wallet.GetDefaultContract()
                     key = self.Wallet.GetKey(contract.PublicKeyHash)
                     if key is not None:
                         print("pubkey %s " % key.PublicKey.encode_point(True))
+                except Exception as e:
+                    print("Exception getting default contract: %s " % e)
```

Comment:
Actually, please remove the whole try..except block. If it breaks it raises an exception anyway, and if it breaks it should not continue like this to the code below.

Reply:
I'm all good with stopping further execution, but same as above: why remove the exception handling? It can also at least catch it, notify instead of printing ~20 lines of stacktrace (they can consult the log file for that) and cleanup + `return`. If you don't catch it, it will leave the wallet in a messed up state i.e. it still prints the keys, address etc, but it will not sync. You can also not `wallet close` it at this point because it will throw
>  File "prompt.py", line 245, in do_close_wallet
>    self._walletdb_loop.stop()
> AttributeError: 'NoneType' object has no attribute 'stop'

Reply:
> It can also at least catch it, notify instead of printing ~20 lines of stacktrace (they can consult the log file for that) and cleanup + return

Yeah, that's also ok!